dash-pipeline/tests/saithrift/pytest/switch/test_saithrift_switch.py

```diff
@@ -6,10 +6,32 @@
        
 @pytest.mark.saithrift
 @pytest.mark.bmv2
-def test_sai_thrift_get_switch_attribute(saithrift_client):
```

Comment:
this is a fix, we should do it in a separate pr.

Reply:
@lguohan we can do that, thanks

Reply:
@lguohan I created https://github.com/Azure/DASH/pull/281 to do this separately. Once #281 is merged, this PR 278 will be synced and get smaller. Thanks for suggestion.

Reply:
Update - once https://github.com/Azure/DASH/pull/281 is merged, we'll sync this PR to main and some of the changes will disappear.